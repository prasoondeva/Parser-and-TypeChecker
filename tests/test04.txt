{
	x : INT;
	y , x : STRING;
	y = 10;
	x = 10;
}