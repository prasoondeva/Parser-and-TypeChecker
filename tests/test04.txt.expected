ERROR CODE 1.1 x